src/Uno.ReferenceImplComparer/Program.cs

```diff
@@ -0,0 +1,124 @@
+					Console.WriteLine($"Validating {identifier}");
+
+					var runtimeAssemblyDefinition = ReadAssemblyDefinition(runtimeAssembly);
+
+					hasErrors |= CompareAssemblies(referenceAssemblyDefinition, runtimeAssemblyDefinition, identifier);
+				}
+			}
+
+			return hasErrors ? 1 : 0;
+		}
+
+		private static bool CompareAssemblies(AssemblyDefinition referenceAssemby, AssemblyDefinition runtimeAssembly, string identifier)
+		{
+			var hasError = false;
+			var referenceTypes = referenceAssemby.MainModule.GetTypes();
+			var runtimeTypes = runtimeAssembly.MainModule.GetTypes().ToDictionary(t => t.FullName);
+
+			foreach(var referenceType in referenceTypes.Where(t => t.IsPublic))
+			{
+				if(referenceType.FullName == "Windows.UI.Xaml.Documents.TextElement")
+				{
+					Console.WriteLine("Skipping Windows.UI.Xaml.Documents.TextElement comparison");
+					continue;
+				}
+
+				if(runtimeTypes.TryGetValue(referenceType.FullName, out var runtimeType))
+				{
+					if(referenceType.BaseType?.FullName != runtimeType.BaseType?.FullName)
+					{
+						Console.WriteLine($"{referenceType.FullName} base type is different {referenceType.BaseType?.FullName} in reference, {runtimeType.BaseType?.FullName} in {identifier}");
+						hasError = true;
+					}
+
+					hasError |= CompareMembers(referenceType.Methods.Where(m => m.IsPublic), runtimeType.Methods, identifier);
+					hasError |= CompareMembers(referenceType.Properties.Where(m => m.GetMethod?.IsPublic ?? false), runtimeType.Properties, identifier);
+					hasError |= CompareMembers(referenceType.Fields.Where(m => m.IsPublic), runtimeType.Fields, identifier);
+					hasError |= CompareMembers(referenceType.Events.Where(m => m.AddMethod?.IsPublic ?? false), runtimeType.Events, identifier);
+				}
+				else
+				{
+					hasError = true;
+					Console.WriteLine($"The type {referenceType} is missing from ");
+				}
+			}
+
+			return hasError;
+		}
+
+		private static bool CompareMembers(IEnumerable<MemberReference> referenceMembers, IEnumerable<MemberReference> runtimeMembers, string identifier)
+		{
+			var hasError = false;
+			var runtimeMembersLookup = runtimeMembers.ToDictionary(m => m.ToString());
+
+			foreach(var referenceMember in referenceMembers)
+			{
+				if (!runtimeMembersLookup.ContainsKey(referenceMember.ToString()))
+				{
+					Console.WriteLine($"The member {referenceMember} cannot be found in {identifier}");
+					hasError = true;
+				}
+			}
+
+			return hasError;
+		}
+
+		private static AssemblyDefinition ReadAssemblyDefinition(string assemblyPath)
+			=> AssemblyDefinition.ReadAssembly(assemblyPath, new ReaderParameters() { AssemblyResolver = new DefaultAssemblyResolver() });
+
+		private static string UnpackArchive(string packagePath)
+		{
+			var path = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString().Trim('{', '}'));
+
+			Directory.CreateDirectory(path);
+
+			Console.WriteLine($"Extracting {packagePath} -> {path}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)